File: app/src/main/java/com/reddittask/android/api/ApiService.java
package com.reddittask.android.api;

import com.reddittask.android.model.GameData;


import retrofit2.http.GET;
import retrofit2.http.Query;
import rx.Observable;

/**
 * Created by marinaracu on 30.11.2017.
 */

public interface ApiService {

    @GET("gaming/top.json?")
    Observable<GameData> getMoviesData(@Query("limit") int limit);
}

File: app/src/main/java/com/keddits/api/ApiService.kt
package com.keddits.api

import com.keddits.model.GameData
import io.reactivex.Single
import retrofit2.http.GET
import retrofit2.http.Query

/**
 * Created by marinaracu on 04.12.2017.
 */
interface ApiService {
    @GET("gaming/top.json?")
    fun getGameData(@Query("limit") limit: Int) : Single<GameData>
}